docs-src/docs/guides/walkthrough.md

```diff
@@ -355,35 +399,432 @@ When you're finished with the example, you can instruct Terraform to destroy the
+```
+
+_above: `guestbook-dpl:2` is in a transitional state_
+
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS      SCALE
+guestbo5fadd  guestbook-svc  demo-env     guestbook-lb  guestbook-dpl:2  2/1
+                                                        guestbook-dpl:1
+```
+
+_above: both versions of the deployment are running at scale_
+
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS       SCALE
+guestbo5fadd  guestbook-svc  demo-env     guestbook-lb  guestbook-dpl:2   1/1
+                                                        guestbook-dpl:1*
+```
+_above: `guestbook-dpl:1` is in a transitional state_
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS      SCALE
+guestbo5fadd  guestbook-svc  demo-env     guestbook-lb  guestbook-dpl:2  1/1
+```
+_above: `guestbook-dpl:1` has been removed, and only `guestbook-dpl:2` remains_
+
+
+---
+
+### Part 7: Prove It
+
+You should now be able to point your browser at the URL for the Guestbook loadbalancer (run `l0 loadbalancer get guestbook-svc` to find it) and see what looks like the same Guestbook application you deployed in the first section of the walkthrough. Go ahead and add a few entries, make sure it's functioning properly. We'll wait.
+
+Now, let's prove that we've actually separated the data from the application by deleting and redeploying the Guestbook application:
+
+`l0 service delete guestbook-svc`
+`l0 loadbalancer delete guestbook-lb`
+
+_(We'll leave the `deploy` intact so we can spin up a new service easily, and we'll leave the environment untouched because it also contained the Redis server.)_
+
+Once those resources have been deleted (you can check the status of your Layer0 jobs with `l0 job list`), let's recreate them.
+
+Create another load balancer:
+
+`l0 loadbalancer create --ports 80:80/http demo-env guestbook-lb`
+
+Create another service, using the **guestbook-dpl** deploy we kept around:
+
+`l0 service create --loadbalancer demo-env:guestbook-lb demo-env guestbook-svc guestbook-dpl:latest`
+
+Wait for everything to spin up, and hit that new load balancer's url (`l0 loadbalancer get guestbook-lb`) with your browser. Your data should still be there!
+
+
+---
+
+### Cleanup
+
+If you're finished with the example and don't want to continue with this walkthrough, you can instruct Layer0 to delete the environment and terminate the application.
+
+`l0 environment delete demo-env`
+
+However, if you intend to continue through [Deployment 3](#deployment-3-guestbook-redis-consul), you will want to keep the resources you made in this section.
 
 
 ---
 
 ## 2b: Deploy with Terraform
 
+As before, we can complete this deployment using Terraform and the Layer0 provider instead of the Layer0 CLI. As before, we will assume that you've cloned the [layer0-examples](https://github.com/quintilesims/layer0-examples) repo and are working in the `iterative-walkthrough/deployment-2/` directory.
```

Comment:
@zpatrick I don't understand why this section should be removed - you're talking about the idea (which I've since removed) of bringing along the `.tfstate` file, I think? This part doesn't mention that file at all, and just says that you can use Terraform instead of Layer0 CLI, which is the whole point of this section. Am I misunderstanding you?